training/culture-and-process/CONTRIBUTING_TO_20_TIME_BACKLOG.md

```diff
@@ -0,0 +1,54 @@
+
+Below is how you can help to get your ideas and resolving bugbears you may have onto the 20% time board to fix or improve our way of working in development.
+
+### Pre-reading
+
+- It is useful to have an understanding of the [Trello board and flow of stories](./TRELLO_BOARD_FLOW.md) and how we use it to organise development workflow as it is mentioned in this documentation
+- It is useful to have an understanding of [70%, 20% and 10% time](./70_20_10_TIME.md)
+
+### How to get ideas onto the 20% Trello Board?
+
+It is beneficial to yourself and others to bounce off ideas that you may think would help the development team before progressing an idea. This can really help develop an idea into something more tangible and allow a larger buy in from the wider team. It could also identify why this might not be a good idea, or there is not a problem to solve here, either way it is still encouraged to voice any ideas and not be disheartened by any push back, remember we are one team and consensus from the team is important.
+
+- It is advisable to reach out to your peers to discuss any ideas, this can be done in any guilds you are part of.
+- If unsure one can raise with a (S7) Senior Developer or Technical Lead.
+
+Once you are confident in your idea/suggestion, you may add your story to [triage column in the 20% backlog board](https://trello.com/b/5G8rf9cm/20-time-backlog).
+
+### What should be included in the 20% story?
+
+Please remember that when a 20% story is triaged and looked at by others that they will need to be able to understand the story so they can clearly make a decision on the value of this story and prioritise it according to the existing backlog of 20% work.
+
+One should use the relevant templates available on the trello board, these can be found in the column at the end of the board by scrolling right.
+
+**Important things to consider when writing up the story:**
+
+- The story should have a clear succinct title
+- Adding the relevant labels on the story (Spike, Bug etc.)
+- A clear description of the problem or suggested improvement
+- Include what cost is this problem having on us and the value of doing this story?
+    - Wasting development time
+    - How much time is sunk into manually fixing/putting in hacks?
+    - Running services when we don’t need them
+- How much effort would be required to do this?
+- Acceptance Criteria
+    - What needs have to be met?
+
+If possible:
+- Attach to available epics if possible (first column in trello board), including epic tag
+- Attach relevant links to docs/pocs that may already exist
+- Contain the services/libraries (repos) that are affected 
+
+### How will Technical Leads prioritise which stories get picked up first?
+
+Prioritising stories should be based on the value of getting something done for the benefit of the development team. The proposed values are the amount of effort required to complete a story and the impact it has on the development team.
```

Comment:
Also maybe worth mentioning that TLs have a regular 20% time triage and prioritization meeting, so these priorities should be getting checked every 2-4 weeks. Just to give people an idea of cadence